server/service/individualServices/SoftwareUpgrade.js

```diff
@@ -137,187 +115,6 @@ async function PromptForBequeathingDataCausesTransferOfListOfApplications(user,
     });
```

Comment:
The forwarding name NewApplicationCausesRequestForOamRequestInformation has been changed in v2.1.1. Kindly check, update and test it once